api/src/main/java/org/pmiops/workbench/api/CohortBuilderController.java

```diff
@@ -89,10 +84,10 @@
     @Override
     public ResponseEntity<CriteriaListResponse> getCriteriaByTypeAndParentId(String type, Long parentId) {
 
-        final List<Criteria> criteriaList = criteriaDao.findCriteriaByTypeLikeAndParentIdOrderBySortOrderAsc(type, parentId);
+        final List<Criteria> criteriaList = criteriaDao.findCriteriaByTypeAndParentId(type, parentId);
```

Comment:
Does the order the criteria come back in matter? (I saw you dropped order by here...)

Reply:
yes.. it was added to the Criteria object using @Query annotation.